config/printer-creality-ender5-s1-2023.cfg

```diff
@@ -0,0 +1,165 @@
+# Creality Ender 5 S1
```

Comment:
As Creality is known to switch around boards, please also provide the exact information of the used board, e.g. "Creality v4.7.11 Rev 38" as indicated on the Silkscreen

Reply:
Will change to 

`Creality Ender 5 S1 (HW Version: CR4NS200141C13)`

Thanks!